### ISSUE TITLE
layer-wise learning rate

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [keras-team/keras#9881]
In latest keras, is there a implementation for set different learning rate for each layer? 
If yes, please give me a example of how to use it. thanks.

**Comment by FirefoxMetzger** [comment]
I might be mistaken, but I think you will get a faster response over at StackOverflow. The issues here are meant for bugs or specific feature requests.

If you are making such a request; please update your question and state what you propose more clearly =)
